front/sssscs/src/login/LoginService.ts

```diff
@@ -0,0 +1,17 @@
+import axios, { AxiosResponse } from "axios";
+
+export interface UserLoginDto {
+    email: string,
+    password: string,
+};
+
+export interface UserLoginResultDto {
+    todo: string,
+};
+
+export class LoginService {
+    static async login(dto: UserLoginDto): Promise<AxiosResponse<UserLoginResultDto>> {
+        const env_url = "http://127.0.0.1:8080";
```

Comment:
Extract environment.ts for url

Reply:
Done in [f8f5546](https://github.com/magley/sssscs/pull/1/commits/f8f55465c6d87638c36cb8e8d2d6f203bb7598f4).